base/sparse/sparsematrix.jl

```diff
@@ -164,7 +164,7 @@ end
 
 function convert{Tv,Ti}(::Type{SparseMatrixCSC{Tv,Ti}}, M::AbstractMatrix)
     m, n = size(M)
-    (I, J, V) = findnz(M)
+    (I, J, V) = isempty(M) ? (Ti[],Ti[],Tv[]) : findnz(M)
```

Comment:
Thanks, will make that change.
I think SparseMatrixCSC breaks for any type that has no `zero` defined.
Not sure if and how to handle that. Should constructor optionally accept a value to be used as `zero`?